src/common/user-store.ts

```diff
@@ -28,6 +28,7 @@ export interface UserPreferences {
   downloadBinariesPath?: string;
   kubectlBinariesPath?: string;
   openAtLogin?: boolean;
+  tablesConfiguration?: Map<string, number>
```

Comment:
This type needs to be `JSON.strignify`-able. Please change this to `Record<K, V>`.